DetectorDescription/DDCMS/plugins/dd4hep/DDDefinitions2Objects.cc

```diff
@@ -2245,8 +2245,6 @@ static long load_dddefinition(Detector& det, xml_h element) {
 
       // Can not deal with reflections without closed geometry
       det.manager().CloseGeometry();
-      // Convert reflections via TGeo reflection factory
```

Comment:
@ghugo83 - you can also delete the previous line. It’s not needed to close the geometry here.
Thanks!

Reply:
@ianna Addressed in cleaning PR https://github.com/cms-sw/cmssw/pull/32241